packages/starlight/translations/tr.json

```
@@ -3,7 +3,7 @@
   "search.label": "Ara",
   "search.shortcutLabel": "(Aramak için / bas)",
   "search.cancelLabel": "İptal",
-  "search.devWarning": "Arama yalnızca üretim derlemelerinde kullanılabilir. \nYerel bilgisayarınızda test etmek için siteyi derleyin ve önizleme yapın.",
+  "search.devWarning": "Arama yalnızca canlı sürümde kullanılabilir. \nYerel bilgisayarınızda test etmek için siteyi derleyin ve önizleme yapın.",
```

Are you sure the previous version is incorrect?

In English the text here is “Search is only available in production builds” and at least according to Google Translate that is what the current version says vs “live version”.

---

If the goal is to translate the word meaning exactly without considering the context, the previous version is correct. However, it conveys the wrong message. The new translation is more understandable as slang used by Turkish developers.

---

There is no translation option for "production builds" in Turkish to complete the meaning perfectly. So we should consider the sentences together to keep meaning as much as possible.

Second sentence points out the "preview" stage. "Üretim derlemeleri" defines the stage while "canlı sürüm" refers to result and ignores the "preview" stage.

---

Thank you for the feedback @erkanaltinors! Sounds like we should stick with the current translation in this case.
```suggestion
  "search.devWarning": "Arama yalnızca üretim derlemelerinde kullanılabilir. \nYerel bilgisayarınızda test etmek için siteyi derleyin ve önizleme yapın.",
```